### ISSUE TITLE
npm install issue with Next.js

### ISSUE THREAD

**Issue by lorenzejay** [ZeroCho/react-filestack#124]
I am having problems installing this package. What can I do here?
<img width="687" alt="Screen Shot 2021-11-02 at 6 30 25 AM" src="https://user-images.githubusercontent.com/63378463/139856422-2d42edb2-0234-4431-92d7-85ba155473ae.png">

**Comment by taw96** [comment]
same for me, @lorenzejay did you find any solution?

**Comment by thomaslenaour** [comment]
Same for me with `create-react-app` (node v16.13.0, npm 8.1.0)

**Comment by lorenzejay** [comment]
> same for me, @lorenzejay did you find any solution?

@taw96  Still none unfortunately

**Comment by taw96** [comment]
@lorenzejay @thomaslenaour npm i -force solved it for me

**Comment by rajkumarvoleti** [comment]
use "npm i filestack-react --leagcy-peer-deps"

**Comment by TahirAli32** [comment]
> @lorenzejay @thomaslenaour npm i -force solved it for me

it works. Thanks

**Comment by ConnorMooneyhanABT** [comment]
Yeah, forcing it works, but a serious project like this should make the effort to upgrade to React 18. They didn't even upgrade to React 17 and they haven't committed to the project since September 2021, so I don't trust them enough to just ignore the versioning difference with a force install.

**Comment by ConnorMooneyhan** [comment]
Yeah, forcing it works, but a serious project like this should make the effort to upgrade to React 18. They didn't even upgrade to React 17 and they haven't committed to the project since September 2021, so I don't trust them enough to just ignore the versioning difference with a force install.
